Platform/CoffeelakeBoardPkg/Library/Stage2BoardInitLib/Stage2BoardInitLib.c

```diff
@@ -971,6 +969,25 @@ BoardInit (
     if (EFI_ERROR (Status)) {
       DEBUG ((DEBUG_WARN, "VBT not found %r\n", Status));
     }
+    if (GetPayloadId () == UEFI_PAYLOAD_ID_SIGNATURE && GetBootMode() != BOOT_ON_S3_RESUME) {
+      //
+      // Update TSEG region with S3 save/restore info
+      //
+      SmmBaseInfo = (SMMBASE_INFO *) (MmioRead32 (TO_MM_PCI_ADDRESS (0x00000000) + TSEG) & ~0xF);
+      SmmSize = (MmioRead32 (TO_MM_PCI_ADDRESS (0x00000000) + BGSM) & ~0xF) - (UINT32)SmmBaseInfo;
+      ZeroMem ((VOID *)SmmBaseInfo, SmmSize);
```

Comment:
Better not have this zero memory. often SMRAM is 8MB or 16MB. it impact performance.
also it makes the S3 lib depends on platform to zero the library.